packages/components/src/components/dropdown/CustomRender.tsx

```diff
@@ -71,6 +71,8 @@ export const Input = (props: any) => {
     onKeyUp={props?.selectProps?.onKeyUp}
     isHidden={inputAlwaysDisplayed ? !inputAlwaysDisplayed : false}
     aria-label={props?.selectProps?.label}
+    aria-describedBy={props?.selectProps?.['aria-describedby']}
+    aria-invalid={props?.selectProps?.['aria-invalid']}
```

Comment:
Why don't we refactor this and spread the entire object ? @aurelie-forzani @antoinerollindev 

```suggestion
    {...props?.selectProps}
```

Reply:
Forget about that, it's too risky